OpenRA.Mods.Common/AudioLoaders/Mp3Loader.cs

```diff
@@ -18,12 +18,37 @@ namespace OpenRA.Mods.Common.AudioLoaders
 {
 	public class Mp3Loader : ISoundLoader
 	{
+		bool IsMp3(Stream s)
```

Comment:
```suggestion
		bool IsMP3(Stream s)
```

Isn't it abbreviation?

Reply:
But we use "Mp3" everywhere?